### DISCUSSION TITLE
How do we know when we have found the crash site?

### DISCUSSION THREAD

**Discussion by connorjak** [SimDynamX/SpaceTeamsPRO#60] [2024 Lunar SAR Competition Q&A] [answered]
(Copied from a student question)
 
We have made some good progress on our code for the Lunar S&R. We are still not sure how we are supposed to check when we have found the crash site. Will space teams automatically stop the program when we have run into the crash site or do we need to run a line of code at every checkpoint to check if we found the crash site?

**Comment by connorjak** [comment, marked as answer]
The sim will freeze (0 timescale) and there will be an alert on screen that tells you the sim was completed and your elapsed time since the sim started.

The end condition for Task 1 is one of the rovers getting to within the 20 meter scan radius of the simulated LIDAR. The CompetitionBackend script constantly checks for this, so your only job for Task 1 is to get a rover to come across the crash site.

**Reply by Ju5t1nL3** [reply]
What is the shape of the scan? Is it a cone?

**Reply by connorjak** [reply]
The LIDAR scan coverage is a sphere
